appeals/web/src/server/appeals/appeal-details/costs/costs.controller.js

```diff
@@ -28,17 +28,19 @@ const renderSelectDocumentType = async (request, response) => {
 	}
 
 	// TODO: BOAT-1168 - add API endpoint to retrieve document type options (hardcoded for now)
+	const { costsCategory } = params;
```

Comment:
Just a tip, you can destructure this directly in the expression above: `onst { errors, currentAppeal, session, params: { costsCategory } } = request;`

Reply:
👍